### ISSUE TITLE
ru_ru lang update

### ISSUE THREAD

**Issue by Remyvi12345** [HexagonNico/ReworkedMetals#16]
[ru.ru.zip](https://github.com/HexagonNico/ReworkedMetals/files/7842275/ru.ru.zip)
Hey! I translated the mod into Russian again! Made some edits, etc.
Current update: 1.18.1
Mod version: 3.0

**Comment by HexagonNico** [comment]
Hi. Sorry, but the language file you sent is exactly the same as the old one.

**Comment by Remyvi12345** [comment]
Oh sorry, I messed up the files, here's the update
[ru_ru.zip](https://github.com/HexagonNico/ReworkedMetals/files/7849068/ru_ru.zip)

**Comment by HexagonNico** [comment]
Thank you! Will be added in the next update!

**Comment by Remyvi12345** [comment]
Oh my God, Hexagon, I'm sorry again, I did not notice a couple of typos in my translation, and not improvements. I fixed everything. Sorry again.. But for that the translation looks much better, I did not make a literal translation - for a Russian person it will look stupid, I made the translation exactly as it would have looked in the original Minecraft translation, so that the names of things would fit into the game.
[ru_ru.zip](https://github.com/HexagonNico/ReworkedMetals/files/7849596/ru_ru.zip)
.
